style.css

```diff
@@ -0,0 +1,138 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: 'Poppins'sans-serif;
```

Comment:
[OPTIONAL]

 The font Poppins in not applying to your texts, you can fix it by change this line to this code
`font-family: 'Poppins', sans-serif;`
Kindly implement this for all the place you have the font-family property

Reply:
All that I can say is Thanks a bunch for this great review